### PR TITLE
33Across: Added Video Support

### PR DESCRIPTION
**Pull request by curlyblueeagle**
## Type of change

- [x] Feature
- [x] Refactoring (no functional changes, no api changes)
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change
- Added Video support
- Refactored code for better organization and readability

**Contact email of the adapter’s maintainer**
headerbidding@33across.com

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/
https://github.com/prebid/prebid.github.io/pull/2441

### CODE REVIEW

File: modules/33acrossBidAdapter.md [hunk 1]
```diff
@@ -10,23 +10,145 @@ Maintainer: headerbidding@33across.com
...
+      sizes: [
+          [300, 250], 
+          [728, 90]
+      ]
+    }   
+  } 
+  bids: [{
+    bidder: '33across',
+    params: {
+        siteId: 'sampleGUID123',
```
Comment (msm0504) [inline modules/33acrossBidAdapter.md:32]: Include valid params in the README that will return data if used in a test page

**Review by msm0504 — CHANGES_REQUESTED** [review]

File: modules/33acrossBidAdapter.md (hunk 1 above)
Comment (curlyblueeagle) [reply modules/33acrossBidAdapter.md]: I have provided a sample GUID that is valid and will help your testing. Please let me know if you need anything more

File: modules/33acrossBidAdapter.js [hunk 2]
```diff
@@ -173,53 +245,187 @@ function _createServerRequest({bidRequest, gdprConsent = {}, uspConsent, pageUrl
...
+  const videoBidderParams = utils.deepAccess(bidRequest, 'params.video', {});
+
+  const videoParams = {
+    ...videoAdUnit,
+    ...videoBidderParams // Bidder Specific overrides
+  };
+
+  const video = {}
+
+  const {w, h} = _getSize(videoParams.playerSize[0]);
```
Comment (curlyblueeagle) [inline modules/33acrossBidAdapter.js:340]: @msm0504 assuming that playerSize although set as an `[int, int]` in Ad Unit definition (per docs) will be sent as `[[int, int]]` which is how I am seeing in `bidRequest`

Comment (msm0504) [reply modules/33acrossBidAdapter.js:340]: That's correct

**Review by msm0504 — APPROVED** [review]

**Pull request merged**